manager/testdata/new-blueprints/scenario2/blueprint.yaml

```diff
@@ -28,6 +28,9 @@ spec:
               bucket: srcbucket
               object: paysim.parq
           dataformat: parquet
+          actions:
```

Comment:
Can you please add a comment how the read-module should know that the `ghcr.io/mesh-for-data/m4d-arrow-flight-transform-conf:0.1.0` chart was chosen to configure this action as a plugin?